File: octo/msgp/msgp-server/src/main/scala/com/sankuai/octo/msgp/model/LogCollectorResult.scala
package com.sankuai.octo.msgp.model

import play.api.libs.json._

/**
 * Created by dreamblossom on 15/9/30.
 */
case class LogCollectorResult(providerAppKey: String, consumer2QpsList: List[ConsumerResult], spanName: String)

object LogCollectorResult {
  implicit val logCollectorResultReader = Json.reads[LogCollectorResult]
  implicit val logCollectorResultWriter = Json.writes[LogCollectorResult]
}

File: octo/oswatch/src/main/java/com/sankuai/octo/oswatch/model/LogCollectorResult.java
package com.sankuai.octo.oswatch.model;

import java.util.List;

/**
 * Created by chenxi on 7/27/15.
 */
public class LogCollectorResult {
    String providerAppKey;
    List<ConsumerResult> consumer2QpsList;
    String spanName;

    public String getProviderAppKey() {
        return providerAppKey;
    }

    public void setProviderAppKey(String providerAppKey) {
        this.providerAppKey = providerAppKey;
    }

    public List<ConsumerResult> getConsumer2QpsList() {
        return consumer2QpsList;
    }

    public void setConsumer2QpsList(List<ConsumerResult> consumer2QpsList) {
        this.consumer2QpsList = consumer2QpsList;
    }

    public String getSpanName() {
        return spanName;
    }

    public void setSpanName(String spanName) {
        this.spanName = spanName;
    }
}

